src/main/groovy/com/bmuschko/gradle/cargo/tasks/local/LocalCargoContainerTask.groovy

```diff
@@ -87,11 +87,11 @@ class LocalCargoContainerTask extends AbstractCargoContainerTask {
     Integer rmiPort
 
     /**
-     * Timeout after which the container start/stop is deemed failed.
+     * Container timeout after which the container start/stop is deemed failed.
```

Comment:
The word "container" is likely to be misunderstood for OCI/Docker containers these days. Can you remove the change?